src/storage/txn/scheduler.rs

```diff
@@ -471,33 +483,30 @@ fn process_read<E: Engine>(
                 None,
                 ctx.get_isolation_level(),
             );
-            let res = match reader.seek_ts(start_ts).map_err(StorageError::from) {
```

Comment:
Is it safe to skip `StorageError::from`?

Reply:
The previous error chain is `StorageError(MvccError(..))` and the new chain is `StorageError(TxnError(MvccError(..)))`. It should be fine because the new chain is what other commands will use and common errors should be handled correctly.